go/libraries/doltcore/doltdb/foreign_key_coll.go

```diff
@@ -75,6 +74,21 @@ func (f *ForeignKeyViolationError) Error() string {
 	}
 }
 
+// getRowValuesAsString returns row values as string format.
+// Variables 'key' and 'val' have both 'tags and values' added in them, so tags are removed
```

Comment:
Are we removing the tags? Unless `IterFields` removes tags, they'll be included. This can be caught in the test you've added, by expanding the message that we're expecting, as right now it's just checking for `"violation"`

Reply:
`IterFields` includes tags.